build-parent/pom.xml

```diff
@@ -677,8 +677,47 @@
                     <artifactId>smallrye-certificate-generator-maven-plugin</artifactId>
                     <version>${smallrye-certificate-generator.version}</version>
                 </plugin>
+                <plugin>
+                    <groupId>com.diffplug.spotless</groupId>
+                    <artifactId>spotless-maven-plugin</artifactId>
+                    <version>2.44.4</version>
+                    <configuration>
+                        <java>
+                            <removeUnusedImports />
+                            <formatAnnotations />
+                            <eclipse>
+                                <file>${maven.multiModuleProjectDirectory}/independent-projects/ide-config/src/main/resources/eclipse-format.xml</file>
+                            </eclipse>
+                            <importOrder>
+                                <file>${maven.multiModuleProjectDirectory}/independent-projects/ide-config/src/main/resources/eclipse.importorder</file>
+                            </importOrder>
+                        </java>
+                    </configuration>
+                    <executions>
+<!--                        <execution>-->
+<!--                            <id>spotless-check</id>-->
+<!--                            <goals>-->
+<!--                                <goal>apply</goal>-->
+<!--                            </goals>-->
+<!--                            <phase>verify</phase>-->
+<!--                        </execution>-->
+                        <execution>
+                            <id>spotless-apply</id>
+                            <goals>
+                                 <goal>apply</goal>
```

Comment:
maven has `apply` as `check` fails the build; then apply is the end result. Might leverage the auto fix capability directly.